eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -90,13 +90,29 @@ function Invoke-GitHubAPIGet {
   return $resp
 }
 
-function SplitMembers ($membersString)
+function Split-GitHubAPIParameterMembers ($membersString)
 {
   if (!$membersString) { return $null }
   return @($membersString.Split(",") | % { $_.Trim() } | ? { return $_ })
 }
 
-function List-PullRequests {
+function Set-GitHubAPIParameters ($members,  $parameterName, $parameters, $allowEmptyMembers=$false)
+{
+  if ($members) { 
+    if ($members -is [array])
+    {
+      $parameters[$parameterName] = @($members)
+    }
+    elseif (![System.String]::IsNullOrWhiteSpace($members) -or $allowEmptyMembers) {
+      $memberAdditions = Split-GitHubAPIParameterMembers -membersString $members
+      $parameters[$parameterName] = @($memberAdditions)
+    }
+  }
+
+  return $parameters
+}
+
+function List-GithubPullRequests {
```

Comment:
For consistency I'd make this `GitHub` instead of `Github`. There are a couple other places you should try to match case as well.